firebase-functions/webhook-router/src/notion/verification.ts

```diff
@@ -0,0 +1,119 @@
+import crypto from "crypto";
```

Comment:
Code is mostly the same as for Slack, but I'm resisting the urge to refactor a common layer.